code/API_definitions/qod-api.yaml

```diff
@@ -341,17 +374,116 @@ components:
       description: |
         Ports may be specified as a list of ranges or single ports.
     QosProfile:
+      description: Data type with attributes of a QosProfile
+      type: object
+      properties:
+        id:
+          type: string
+          example: foo-bar
+          description: Unique gererated ID for a QoD Profile
+          minLength: 3
+          maxLength: 64
+          format: string
+        name:
+          type: string
+          example: foo-bar
+          description: Descriptive name for QoD Profile
+          minLength: 3
+          maxLength: 64
```

Comment:
I like having this max constraint (and the min too). I was thinking maybe 64 was too short, but it's probably fine.